redaxo/src/core/default.config.yml

```diff
@@ -10,6 +10,12 @@ fileperm: '0664'
 dirperm: '0775'
 session_duration: 7200
 session_keep_alive: 21600
+backend_login_policy:
+    login_tries_1: 3
+    relogin_delay_1: 5
+    login_tries_2: 50
+    relogin_delay_2: 3600
```

Comment:
Was mir gerade noch einfällt: Statt dieser statischen 2 Stufen, hätte man es ggf. auch dynamischer machen können. Also irgendwie in einem Array mit beliebig vielen Stufen.
Aber für mich ist es auch ok so. Muss ggf. @dergel sonst nochmal sagen, wie viel Flexibilität er benötigt.

Reply:
Hab die bestehende logik 1:1 übernommen. Das issue von jan, stellt meiner meinung in Frage ob man überhaupt 2 brauch oder eine reicht.

daher bin ich den weg gegangen es zu lassen wie es ist

Reply:
Ich denke, dass wir diese 2. Stufe gar nicht brauchen. Gefühlt, ist das ein Überbleibsel und seltsame Regelung von früher. Meinetwegen könnten wir auch ganz auf den Delay verzichten. Sehe da keinen Vorteil.

Reply:
Vorhin mit Jan drüber gesprochen. Und denke stimme ihm zu. Eigentlich reicht auch einfach eine Grenze, ab der man direkt gesperrt wird (z.b. nach 20 Versuchen, oder wie man es halt einstellt).

Reply:
Ich sehe keinen grund warum man die bestehende logik anpassen sollte. 

Wollen wir es dennoch tun ?

Reply:
Ich sehe da schon Gründe, die dafür sprechen. Wir führen ja frisch die Konfiguration ein. Und das Konstrukt mit den vier Werten ist nicht allzu intuitiv. Da wäre eine simplere Einstellung (ein Wert) schon schöner. Vor allem, wenn wir denken, dass die Flexibilität mit den 4 Werten gar nicht benötigt wird.
Und nebenbei würden wir auch allgemein den Code etwas entschlacken, weil die Logik mit den Zeitsperren entfallen würde.

Aber selbstverständlich gibt es auch Gründe für die aktuelle Variante: Sie funktioniert seit Jahren, etc.

Letztlich ist beides für mich ok.

Reply:
Ich würde hier erstmal 1:1 mergen und die vereinfachung als separaten PR machen

Reply:
here we go https://github.com/redaxo/redaxo/pull/5200